lib/devcycle-ruby-server-sdk/eval_reasons.rb

```diff
@@ -0,0 +1,71 @@
+module DevCycle
+  # Default reasons for evaluation failures
+  module DEFAULT_REASONS
```

Comment:
simplify this definition to only include the ones we are using in the SDK, its should be limited to:
- missing config
- user not targeted
- error